sources/niUI/src/WidgetTree.cpp

```diff
@@ -689,7 +726,7 @@ class cWidgetTreeNode : public ni::cIUnknownImpl<ni::iWidgetTreeNode,ni::eIUnkno
             mTextRect.Move(Vec2<tF32>(mptrWidget->GetSize().x+margin.x+margin.z,0));
           }
           else {
-            newPos.x = treePos.x+nodeRect.Left()+mTextRect.GetWidth();
+            newPos.x = ptrParentWidget->mpWidget->GetAbsoluteRect().GetRight() - mptrWidget->GetSize().x - margin.z;
```

Comment:
I read the commit description and see the code here, but I'm not 100% sure what it does. Does is change the default layout? or does it fix what you'd consider a bug? Possible to share a screencap of the change?